src/System/SysInfo.hsc

```diff
@@ -120,3 +124,9 @@ sysInfo = do
       free sptr
       err <- getErrno
       return $ Left err
+#else
+-- | Functor for getting system information. On non-Lonux sytems always fails
```

Comment:
Fixed the Linux typo. :) Have released a new version in Hackage with your changes.